frontend/src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect, useCallback } from "react";
+import "./App.css";
+import SearchBar from "./components/SearchBar.jsx";
+import NewBoardModal from "./components/NewBoardModal.jsx";
+import KudosList from "./components/KudosList.jsx";
+
+const App = () => {
+  const [isNewBoardModalVisible, setNewBoardModalVisibility] = useState(false);
+  const [visibleBoards, setVisibleBoards] = useState([]);
+
+  const getBoards = useCallback(async () => {
+    const response = await fetch("http://localhost:3000/boards");
+    const data = await response.json();
+    setVisibleBoards(data);
+  });
```

Comment:
Why `useCallback` without a dependency array? This function will be recreated on every render.